python/nvstrings.py

```diff
@@ -339,6 +339,18 @@ def get_cpointer(self):
         """
         return self.m_cptr
 
+    def create_from_ipc(self):
```

Comment:
This should not be a member function. The function should not require an existing nvstrings instance to create a new unrelated instance. There are functions like this near the top of the source file.
Also, where is the ipc object (list obj)? Should that not be a parameter?